app/frontend/styles/components/_tabs.scss

```diff
@@ -0,0 +1,64 @@
+.tabs-component {
+  margin-bottom: govuk-spacing(4);
```

Comment:
Do you think we could override this to 0? As it's creating unnecessarily huge gap between the navigation and the applications
<img width="664" alt="Screenshot 2024-02-16 at 14 40 53" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/136708005/e1ab5188-629f-4e7c-87f9-e63243703abd">

Reply:
Yes, I will do

Reply:
Done!